### PR TITLE
Add config for length player name

### PR DESCRIPTION
**Pull request by ramon-bernardo**
### Pull Request Prelude

- [x] I have followed [proper The Forgotten Server code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed
- Add config for lenght player name
- Now we can easily change to 25 and have 12.x support

[cont]: https://github.com/otland/forgottenserver/wiki/Contributing
[code]: https://github.com/otland/forgottenserver/wiki/TFS-Coding-Style-Guide

### CODE REVIEW

File: src/game.cpp [hunk 1]
```diff
@@ -506,7 +506,7 @@ Player* Game::getPlayerByGUID(const uint32_t& guid)
 ReturnValue Game::getPlayerByNameWildcard(const std::string& s, Player*& player)
 {
 	size_t strlen = s.length();
-	if (strlen == 0 || strlen > 20) {
+	if (strlen == 0 || strlen > PLAYER_NAME_LENGHT) {
```
Comment (EPuncker) [inline src/game.cpp:509]:
```suggestion
	if (strlen == 0 || strlen > PLAYER_NAME_LENGTH) {
```

File: src/game.cpp [hunk 2]
```diff
@@ -3228,7 +3228,7 @@ void Game::playerSetFightModes(uint32_t playerId, fightMode_t fightMode, bool ch
 
 void Game::playerRequestAddVip(uint32_t playerId, const std::string& name)
 {
-	if (name.length() > 20) {
+	if (name.length() > PLAYER_NAME_LENGHT) {
```
Comment (EPuncker) [inline src/game.cpp:3231]:
```suggestion
	if (name.length() > PLAYER_NAME_LENGTH) {
```

File: src/game.h [hunk 3]
```diff
@@ -63,6 +63,8 @@ enum GameState_t {
 	GAME_STATE_MAINTAIN,
 };
 
+static constexpr int32_t PLAYER_NAME_LENGHT = 20;
```
Comment (EPuncker) [inline src/game.h:66]:
```suggestion
static constexpr int32_t PLAYER_NAME_LENGTH = 25;
```

**Review by EPuncker — APPROVED** [review]

**Comment by Zbizu** [comment]
there doesn't seem to be any real limit to names length from client side
obraz

I wasn't able to log in as a character with more than 70 chars in name though.

**Pull request merged**